login/meson.build

```diff
@@ -44,15 +46,21 @@ glome_login = executable(
     'glome-login',
     [
         'main.c',
-        'config.c',
-        'config.h',
     ],
-    dependencies : confuse_dep,
     link_with : login_lib,
     include_directories : glome_incdir,
     install : true,
     install_dir : get_option('sbindir'))
 
+cc = meson.get_compiler('c')
+libpam = cc.find_library('pam')
+pam_glome = shared_library('pam_glome',
+    'pam.c',
+    dependencies : [libpam],
+    link_with : [glome_lib, login_lib],
+    include_directories : glome_incdir,
+    install : true)
```

Comment:
Please add a TODO to install this into the correct directory or add the support for that. I know it's messy, at least last time I looked at how to solve this there was no pre-built support in meson.

Reply:
Added support to install as `$libdir/security/pam_glome.so`